fiftyone/utils/ptv3.py

```
@@ -0,0 +1,404 @@
+        )
+        self.point_cloud_range = self.parse_array(
+            d, "point_cloud_range", default=None
+        )
+        if (
+            self.point_cloud_range is not None
+            and len(self.point_cloud_range) != 6
+        ):
+            raise ValueError(
+                "point_cloud_range must have 6 values "
+                "[xmin, ymin, zmin, xmax, ymax, zmax]; found %s"
+                % (self.point_cloud_range,)
```

Ruff (0.15.17)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)